SpatialGDK/Source/SpatialGDK/Public/Interop/ActorSystem.h

```diff
@@ -122,6 +133,10 @@ class ActorSystem
 	void RefreshEntity(const Worker_EntityId EntityId);
 	void ApplyFullState(const Worker_EntityId EntityId, USpatialActorChannel& EntityActorChannel, AActor& EntityActor);
 
+	void InvokeRepNotifies();
```

Comment:
I would maybe put a comment about the member-but-not-really-member variable RepNotifiesToSend either here, or in the cpp or somewhere, but this may just be my caution speaking. I just feel like the link may be a bit weak otherwise.

Reply:
yee, added one